### PR TITLE
Remove/rankers

### PR DESCRIPTION
**Pull request by vaskonov**
Remove/rankers

### CODE REVIEW

File: deeppavlov/configs/ranking/rel_ranking.json [hunk 1]
```diff
@@ -1,88 +0,0 @@
-{
```
Comment (puleon) [inline deeppavlov/configs/ranking/rel_ranking.json:1]: Please ask Dmitry Evseev about this file

Comment (vaskonov) [reply deeppavlov/configs/ranking/rel_ranking.json:1]: @dmitrijeuseew

File: deeppavlov/core/common/registry.json [hunk 2]
```diff
@@ -10,9 +10,6 @@
   "bert_ranker": "deeppavlov.models.bert.bert_ranker:BertRankerModel",
   "bert_ranker_preprocessor": "deeppavlov.models.preprocessors.bert_preprocessor:BertRankerPreprocessor",
   "bert_sep_ranker": "deeppavlov.models.bert.bert_ranker:BertSepRankerModel",
-  "bert_sep_ranker_predictor": "deeppavlov.models.bert.bert_ranker:BertSepRankerPredictor",
-  "bert_sep_ranker_predictor_preprocessor": "deeppavlov.models.preprocessors.bert_preprocessor:BertSepRankerPredictorPreprocessor",
-  "bert_sep_ranker_preprocessor": "deeppavlov.models.preprocessors.bert_preprocessor:BertSepRankerPreprocessor",
```
Comment (puleon) [inline deeppavlov/core/common/registry.json:15]: You left BertRankerPreprocessor but you remove BertSepRankerPreprocessor, why?

**Review by puleon — CHANGES_REQUESTED** [review]

File: deeppavlov/configs/ranking/ranking_ubuntu_v2_bert_sep.json [hunk 3]
```diff
@@ -1,72 +0,0 @@
-{
```
Comment (IgnatovFedor) [inline deeppavlov/configs/ranking/ranking_ubuntu_v2_bert_sep.json:1]: used as example at 32 line on neural_ranking.rst
listed at features/overview.rst 183 line

File: deeppavlov/configs/ranking/ranking_ubuntu_v2_mt_word2vec_smn.json [hunk 4]
```diff
@@ -1,127 +0,0 @@
-{
```
Comment (IgnatovFedor) [inline deeppavlov/configs/ranking/ranking_ubuntu_v2_mt_word2vec_smn.json:1]: neural_ranking.rst, 52 line
features/overview.rst 177 and 215 lines

File: docs/features/models/neural_ranking.rst [hunk 5]
```diff
@@ -14,14 +14,6 @@ BERT Ranking
 
 Before using models make sure that all required packages are installed running the command for TensorFlow:
```
Comment (IgnatovFedor) [inline docs/features/models/neural_ranking.rst:15]:
```suggestion
Before using models make sure that all required packages are installed running the command:
```

**Review by IgnatovFedor — CHANGES_REQUESTED** [review]

File: docs/features/models/bert.rst [hunk 6]
```diff
@@ -195,7 +176,6 @@ the :doc:`config </intro/configuration>` file must be changed to match new BERT
 
 .. [1] Kuratov, Y., Arkhipov, M. (2019). Adaptation of Deep Bidirectional Multilingual Transformers for Russian Language. arXiv preprint arXiv:1905.07213.
 .. [2] Arkhipov M., Trofimova M., Kuratov Y., Sorokin A. (2019). `Tuning Multilingual Transformers for Language-Specific Named Entity Recognition <https://www.aclweb.org/anthology/W19-3712/>`__ . ACL anthology W19-3712.
-.. [3] McDonald, R., Brokos, G. I., & Androutsopoulos, I. (2018). Deep relevance ranking using enhanced document-query interactions. arXiv preprint arXiv:1809.01682.
 .. [4] Yanran Li, Hui Su, Xiaoyu Shen, Wenjie Li, Ziqiang Cao, and Shuzi Niu. DailyDialog: A Manually Labelled Multi-turn Dialogue Dataset. IJCNLP 2017.
```
Comment (puleon) [inline docs/features/models/bert.rst:179]: Fix enumeration 4->3 etc.

**Review by puleon — APPROVED** [review]

**Review by IgnatovFedor — APPROVED** [review]

**Review by IgnatovFedor — CHANGES_REQUESTED** [review]
rel_ranking.json is used by kbqa configs and mentioned in the kbqa.rst

**Review by IgnatovFedor — APPROVED** [review]

**Pull request merged**